qiskit_metal/tests/test_draw.py

```diff
@@ -561,6 +583,30 @@ def test_draw_vector_angle_between(self):
         actual = vector.angle_between([0, 10], [15, 20])
         self.assertAlmostEqualRel(actual, expected, rel_tol=1e-3)
 
+    def test_draw_vec3d_angle_between(self):
+        """Test angle_between in the Vec3D class in utility.py"""
+        vec3d = Vec3D()
+
+        expected = np.pi / 2
+        actual = vec3d.angle_between([1, 0, 0], [0, 1, 0])
```

Comment:
Which method is being exectued?  Is it from the 3D class, are the numbers as expected?

Reply:
Yes, the method in the 2D class is flexible as it just uses numpy methods to do the calculation and hence no need to update it for 3D, and we can use the same thing. Will update the docstring in 2D class.